app/components/web-components/au-checkbox.hbs

```diff
@@ -1,5 +1,6 @@
 <label class="auk-checkbox">
   <input class="auk-checkbox__toggle" type="checkbox" name={{@name}} checked={{@checked}} onChange={{@toggle}}/>
   {{@label}}
+  {{yield}}
```

Comment:
In het geval er een block meegegeven wordt, zou ik `@label` niet meer renderen in de template van `AuCheckbox` en verwachten dat dat verwerkt is in het block (met eventueel custom styling of zo)

Reply:
Moet ik dan niet alle checkboxen aanpassen?

Reply:
Just making sure

Reply:
Nee, zoals hierboven gezegd kan je met de `has-block` helper in de template checken of de gebruiker al dan niet een block meegegeven heeft. Indien er geen meegegeven is, heb je het oude gedrag dat enkel `@label` getoond wordt. 
Vb. https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/d4a42422841871c3c177772d5c474f949b8fdc92/app/components/web-components/vl-alert.hbs#L37-L46

Reply:
Okay got it. Merci :)